lib/components/MusicScreen/music_screen_tab_view.dart

```diff
@@ -185,42 +185,53 @@ class _MusicScreenTabViewState extends State<MusicScreenTabView>
   // If clicked in the # element, it goes to the first one ( pixels = 0 )
   void scrollToLetter(String? clickedLetter) async {
     String? letter = clickedLetter ?? letterToSearch;
-    bool reversed = FinampSettingsHelper
-            .finampSettings.tabSortOrder[widget.tabContentType] ==
-        SortOrder.descending;
     if (letter == null || letter.isEmpty) return;
 
     letterToSearch = letter;
-    var letterCodePoint = letterToSearch!.toLowerCase().codeUnitAt(0);
+    var letterToScrollTo = letterToSearch!.toLowerCase()[0];
+
+    const scrollDuration = Duration(milliseconds: 750);
 
     if (letter == '#') {
-      letterCodePoint = 0;
-    }
-    for (var i = 0; i < _pagingController.itemList!.length; i++) {
-      var itemCodePoint =
-          _pagingController.itemList![i].nameForSorting!.codeUnitAt(0);
-      var diff = itemCodePoint - letterCodePoint;
-      if (reversed ? diff <= 0 : diff >= 0) {
-        timer?.cancel();
-        if (reversed ? diff < 0 : diff > 0) {
-          await controller.scrollToIndex(i,
-              duration: const Duration(milliseconds: 200),
-              preferPosition: AutoScrollPosition.middle);
-        } else {
+      await controller.scrollToIndex(0,
+        duration: scrollDuration,
+        preferPosition: AutoScrollPosition.begin);
+      return;
```

Comment:
Oh, whoops. I'll fix it tomorrow 👀

Reply:
It seems like this is glitching in reverse mode anyway, even when I use your original version. It just scrolls to Z, and then loads duplicate pages, so that the entire list is Z-A and then Z-A again...

Reply:
I noticed issues with getting stuck due to lowercase letters being on top in the reversed mode, but I haven't seen duplicate page loads with the latest code.  There's counter variables tracking both refreshes and pages with the goal of preventing duplicate page loads from going through.  Do you have any more details about this, because I haven't seen it.

Reply:
I'll see if I can give you a repro in a few minutes...

Reply:
Okay, I got two videos of the issue for you. The first one is when using your original fix from #673, the second one is with the version I commited to `redesign` earlier.  
As you can see, when going to the "#" letter, all pages are loaded again for some reason, and scrolling to any letter afterwards is really laggy. And there are actually duplicate pages, as you can see at the end when it scrolls to the letter "E". There is a spot where the last track from "Z" is right before the first track from "A".

Your original fix:

https://github.com/jmshrv/finamp/assets/18015147/6689c693-7c99-4765-aa02-33b2f0ecd0a0

Currently committed fix (basically the same, just with adaptive scroll duration):

https://github.com/jmshrv/finamp/assets/18015147/293c2ad4-b3c5-418c-98c2-8ca3be2a9f89

Reply:
The second video seems to have corrupted, but in the first video, note that the previous a tracks are all lower case, and the following Z tracks are uppercase.  This is just jellyfin's song sorting being case sensitive.  So you select # and it scrolls through the whole song library to the bottom, and then you select a letter and it scrolls through the whole song library to the small cluster of lowercase letters at the top, because it finds those first.

Reply:
Oh, I never even noticed that!  
So that means the fast scroller is completely broken to begin with, if items are sorted based on casing too? Maybe we should submit an issue to the server repo in that case? I can't believe this is working as intended, if the goal is alphanumeric sorting...

Reply:
It's sort of broken, yeah.  Although in most cases the vast majority of songs start with an upper case letter so it doesn't matter that much.  This is a song-specific thing, everything else sorts as expected.  The best solution for now is probably to make the scroller specifically target uppercase letters when in the songs tab, and make nameForSorting return normal casing for songs to match the server.

Reply:
I think we can leave it like it is for now, especially since it's a minor issue. I'd like to keep Finamp's behavior consistent with the web client as much as possible, and not include too much custom logic. I've created an issue at jellyfin/jellyfin#11320